tests/integrations/realtiup/download_binaries.sh

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env bash
+
+# download_binaries.sh will
+# * download all the binaries you need for integration testing
+
+# Notice:
+# Please don't try the script locally,
+# it downloads files for linux platform. We only use it in docker-compose.
+
+set -o errexit
+set -o pipefail
+
+# Specify which branch to be utilized for executing the test, which is
+# exclusively accessible when obtaining binaries from
+# http://fileserver.pingcap.net.
+branch=${1:-master}
```

Comment:
In order to be able to download the required binary in subsequent release branches, how about assign a value to the branch through an environment variable.In this way, we can easily specify environment variables in the CI or local environment to download third-party dependencies of specific branches.
```suggestion
branch=${THIRD_PARTY_TARGET_BRANCH:-master}
```